tests/RobotFramework/tests/cumulocity/custom_operation/custom_operation_multiple_args/operation_multiple_arguments_in_command.robot

```diff
@@ -0,0 +1,26 @@
+*** Settings ***
+Resource            ../../../../resources/common.resource
+Library             Cumulocity
+Library             ThinEdgeIO
+
+Test Setup          Custom Setup
+Test Teardown       Get Logs
+
+Test Tags           theme:c8y    theme:operation    theme:custom
+
+
+*** Test Cases ***
+Run the custom operation with multiple arguments
+    ${operation}=    Cumulocity.Create Operation    description=do something    fragments={"c8y_Command":{"text":""}}
+    Operation Should Be SUCCESSFUL    ${operation}
+    Should Be Equal    ${operation.to_json()["c8y_Command"]["result"]}    command 1\n
```

Comment:
```suggestion
    Should Be Equal    ${operation}.to_json()["c8y_Command"]["result"]    command 1\n
```
Not really a request for change but I'm just surprised that the current syntax is working.

Reply:
No the current syntax is correct...RobotFramework variables use the syntax `${    }` to declare that it is a variable. Inside the this syntax is straight python code, `operation.to_json()["c8y_Command"]["result"]`